.github/pr-commenter.yml

```diff
@@ -0,0 +1,67 @@
+comment:
+  on-update: recreate
+  header: |
+    Thank you for contributing to `exercism/elixir` :purple_heart: :tada:. This is an automated PR comment :robot: for the **maintainers** of this repository that helps with the PR review process. You can safely ignore it and wait for a maintainer to review your changes.
+
+    Based on the files changed in this PR, it would be good to pay attention to the following details when reviewing the PR:
+
+  footer: |
+    ---
+    Automated comment created by [PR Commenter](https://github.com/angelikatyborska/pr-commenter-action) :robot:.
+
+  snippets:
+    - id: any-file-changed
+      files:
+        - '*'
+        - '**/*'
+      body: |
+        - **General steps**
+            - :trophy: Does this PR need to receive a label with a modifier (`reputation/contributed_code/{minor,major}`)? (A medium reputation amount is awarded by default, see [docs](https://github.com/exercism/docs/blob/main/product/reputation.md#4-creating-pull-requests))
+
+    - id: any-exercise-changed
+      files:
+        - 'exercises/**/*'
+      body: |
+        - **Any exercise changed**
+            - :bust_in_silhouette: Does the author of the PR need to be added as an author or contributor in `<exercise>/.meta/config.json` (see [docs](https://github.com/exercism/docs/blob/main/building/tracks/practice-exercises.md#file-metaconfigjson))?
```

Comment:
I'd be a fan of putting the actual emojis in rather than the GH placeholders. I find it harder to reason about `bust_in_silhouette` than 👤 (I had to type it in the comment to work out what it was). But there may be a reason that's a bad idea that I've not thought of?

Reply:
Done! I am not aware of any reasons against doing that.